libportal/portal-qt5.cpp

```diff
@@ -69,3 +70,248 @@ xdp_parent_new_qt (QWindow *window)
   parent->data = (gpointer) window;
   return parent;
 }
+
+namespace XdpQt {
+
+GetUserInformationResult
+getUserInformationResultFromVariant(GVariant *variant)
+{
+    GetUserInformationResult result;
+
+    g_autofree gchar *id = nullptr;
+    g_autofree gchar *name = nullptr;
+    g_autofree gchar *image = nullptr;
+
+    if (variant) {
+        if (g_variant_lookup(variant, "id", "s", &id)) {
+            result.id = id;;
```

Comment:
What's with the double colons?

Reply:
A mistake made once, then copied multiple times.